02/app.js

```diff
@@ -1,3 +1,3 @@
 console.log('Działa! Gratuluję uruchomienia JS z poziomu Node.js!');
-console.log('Moja wersja Node.js to: [tutaj wpisz swoją wersję]');
-console.log('Moja wersja npm to: [tutaj wpisz swoją wersję]');
\ No newline at end of file
+console.log('Moja wersja Node.js to: v18.17.1');
```

Comment:
Czasami jest problem z wersjami node czy npm - na niektórych systemach coś może nie działać np. w wepbacku bo jest wersja, która ma błąd. Jeśli nie widać innych problemów to warto zaproponować aktualizację do najnowszej wersji. Na starszych mack-ach lub linuxach czasami był z tym problem. Trzeba wymusić nowszą wersję.

https://blog.hubspot.com/website/update-node-js
https://askubuntu.com/a/480642